recipe/meta.yaml

```diff
@@ -213,6 +229,14 @@ outputs:
 
 
   - name: tensorflow
+    build:
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+
+      # weigh down cpu implementation and give cuda preference
+      track_features:
+        - tensorflow-cpu          # [cuda_compiler_version == "None"]
```

Comment:
We did do this the other way around normally, esp @isuruf pointed out that we should give CPU builds a higher priority.

Reply:
I am following pytorch here ... https://github.com/conda-forge/pytorch-cpu-feedstock/blob/b8824904342b8ea142d297668bf0aa32790091a6/recipe/meta.yaml#L148-L149

The pip package also gives the GPU version by default.

Reply:
Also you'd only get the GPU version if there is actually a working CUDA driver installation on the system (the `__cuda` virtual package is required by `cudatoolkit`). In that case I think we can argue that it's also desired to get the GPU version by default.

Reply:
Makes sense, I would still like to have @isuruf chime in here.

Reply:
+1 for weighing down CPU, preferring GPU (where possible)

Reply:
@xhochy, since the tensorflow builds are new and @wolfv says that the GPU package falls back to CPU if there's no GPU, it's okay to have GPU as a priority.

Reply:
Thanks!